app/helpers/serialize_intervention_helper.rb

```diff
@@ -0,0 +1,25 @@
+module SerializeInterventionHelper
+  def serialize_intervention(intervention)
+    {
+      id: intervention.id,
+      name: intervention.name,
+      comment: intervention.comment,
+      goal: intervention.goal,
+      start_date: intervention.start_date.strftime('%B %e, %Y'),
+      end_date: intervention.end_date.try(:strftime, '%B %e, %Y'),
+      educator_email: intervention.educator.try(:email),
+      progress_notes: intervention.progress_notes.order(created_at: :asc).map do |progress_note|
```

Comment:
For now I kept the time rendering and sorting on the server.  But this logic also needs to be on the client so the JS code knows where to add a new intervention or progress note in the list, so I think it'd be a good separate improvement to serialize timestamps to the UI and let the UI own this responsibility.